inspirehep/modules/orcid/tasks.py

```diff
@@ -22,18 +22,12 @@
 
 from celery import shared_task
 from celery.utils.log import get_task_logger
-
 from flask import current_app
-
 from invenio_db import db
-
-from invenio_oauthclient.models import RemoteAccount, UserIdentity
-
+from invenio_oauthclient.models import RemoteAccount, RemoteToken, UserIdentity
 from invenio_pidstore.resolver import Resolver
-
 from invenio_search import current_search_client
 from invenio_search.utils import schema_to_index
-
 from requests import RequestException
```

Comment:
`isort` is bad, because it uses alphabetical order rather than a semantical ordering (stdlib/vendored/invenio/inspire/relative).